tests/system/providers/amazon/aws/example_bedrock.py

```diff
@@ -37,30 +81,84 @@
 ) as dag:
     test_context = sys_test_context_task()
     env_id = test_context["ENV_ID"]
+    bucket_name = f"{env_id}-bedrock"
+    input_data_s3_key = f"{env_id}/train.jsonl"
```

Comment:
`json1` is a weird file name, is that a typo?

Reply:
Good catch, but the filename came straight out of the Bedrock docs page.  I hadn't hard of it either and thought it was odd as well, but Google said it is a known file format and it works in manual testing.  So I'm reasonably sure it's not a typo.

```
The key difference [between json and jsonl] is in how they handle multiple JSON objects. Regular JSON files are typically a single, self-contained structure, while JSON Lines use a line-by-line format, allowing for easier streaming and processing of individual objects.
```